src/dto/create-game-request.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { IsBoolean, IsNotEmpty } from "class-validator";
+
+export class CreateGameRequestDto {
+    @IsNotEmpty()
```

Comment:
are those `@IsNotEmpty()` needed?

Reply:
yes I want them all as a must parameters

Reply:
@IsBoolean() does this